src/aliceVision/sensorDB/cameraSensors.db

```diff
@@ -7297,6 +7297,7 @@ Xiaomi;Xiaomi Mi 9 Pro;6.4;kimovil
 Xiaomi;Xiaomi Mi 9 Pro 5G;6.4;kimovil
 Xiaomi;Xiaomi Mi 9 SE;6.4;kimovil
 Xiaomi;Xiaomi Mi 9T Pro;6.4;kimovil
+Xiaomi;Xiaomi Mi 11T;8.42;lapo-luchini
```

Comment:
The idea of the last field is to have the name of the source of the information, in your case https://www.gsmarena.com/.
```suggestion
Xiaomi;Xiaomi Mi 11T;8.42;gsmarena
```